aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -1,47 +1,26 @@
 ---
 remote_endpoint: ~
-name: "v1.11"
+name: "v1.12"
 proposals:
   - name: step_1_increase_max_txn_gas
     metadata:
       title: "Increase max txn gas temporarily for framework upgrade"
       description: "Increase max txn gas temporarily for framework upgrade"
     execution_mode: MultiStep
     update_sequence:
-      - DefaultGasWithOverrideOld:
+      - DefaultGasWithOverride:
+          feature_version: 16
           overrides:
             - name: "txn.max_execution_gas"
               value: 3676000000
   - name: step_2_upgrade_framework
     metadata:
-      title: "Multi-step proposal to upgrade mainnet framework to v1.11"
-      description: "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.11"
+      title: "Multi-step proposal to upgrade mainnet framework to v1.12"
+      description: "This includes changes in https://github.com/aptos-labs/aptos-core/commits/aptos-release-v1.12"
     execution_mode: MultiStep
     update_sequence:
       - Framework:
           bytecode_version: 6
           git_hash: ~
-      - RawScript: aptos-move/aptos-release-builder/data/proposals/randomness_framework_initialization.move
```

Comment:
have we updated the framework upgrade test baseline from 1.10.x to 1.11? otherwise this is still needed.

Reply:
@zjma , the same here. Can you see if the baseline version is bumped up properly in this PR?

Reply:
looks good. i'm just not sure whether this needs to be here or in the target branch to make sure the framework upgrade test pick it up